sechub-pds/src/test/java/com/daimler/sechub/pds/job/PDSFileUploadJobServiceTest.java

```diff
@@ -40,195 +40,240 @@
         /* prepare */
-        String result = "content data";
+        String result = CONTENT_DATA;
         MockMultipartFile multiPart = new MockMultipartFile("file", result.getBytes());
         String fileName = "1234567890123456789012345678901234567890";
-        assertEquals(40,fileName.length());// check precondition
+        assertEquals(40, fileName.length());// check precondition
         job.setState(PDSJobStatusState.READY_TO_START);
-        
+
+        PDSNotAcceptableException exception = assertThrows(PDSNotAcceptableException.class, () -> {
+
+            /* execute */
+            serviceToTest.upload(jobUUID, fileName, multiPart, ACCEPTED_CHECKSUM);
+        });
+
         /* test */
-        expected.expect(PDSNotAcceptableException.class);
-        expected.expectMessage("accepted is only:[CREATED]");
+        assertTrue(exception.getMessage().contains("accepted is only:[CREATED]"));
 
-        /* execute */
-        serviceToTest.upload(jobUUID, fileName, multiPart, ACCEPTED_CHECKSUM);
-        
     }
 
     @Test
-    public void upload_containing_filename_length_41_throws_illegal_argument_exception() {
+    void upload_fails_when_containing_filename_length_41_throws_illegal_argument_exception() {
         /* prepare */
-        String result = "content data";
+        String result = CONTENT_DATA;
         MockMultipartFile multiPart = new MockMultipartFile("file", result.getBytes());
         String fileName = "12345678901234567890123456789012345678901";
-        assertEquals(41,fileName.length());// check precondition
-        
-        /* test */
-        expected.expect(IllegalArgumentException.class);
-        expected.expectMessage("40");
+        assertEquals(41, fileName.length());// check precondition
 
-        /* execute */
-        serviceToTest.upload(jobUUID, fileName, multiPart, ACCEPTED_CHECKSUM);
-        
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> {
+
+            /* execute */
+            serviceToTest.upload(jobUUID, fileName, multiPart, ACCEPTED_CHECKSUM);
+        });
+
+        /* test */
+        assertTrue(exception.getMessage().contains("40"));
     }
-    
+
     @Test
-    public void upload_containing_filename_with_slash_throws_illegal_argument_exception() {
+    public void upload_fails_when_containing_filename_with_slash_throws_illegal_argument_exception() {
         /* prepare */
-        String result = "content data";
+        String result = CONTENT_DATA;
         MockMultipartFile multiPart = new MockMultipartFile("file", result.getBytes());
         String fileName = "123456789/123456789012345678901234567890";
-        assertEquals(40,fileName.length());// check precondition
-        
-        /* test */
-        expected.expect(IllegalArgumentException.class);
-        expected.expectMessage("[a-zA-Z");
+        assertEquals(40, fileName.length());// check precondition
```

Comment:
Is the `// check precondition` really necessary? The comment adds no value, in my opinion. I think it is the result of copy and past.

Reply:
The precondition checks seems to be more confusing than helpful. They were made to ensure the literals were correct when developing the tests. But the literals do not change any longer but are static and work.

So I completely removed all precondition checks of
``` 
 assertEquals(40, fileName.length());// check precondition
```
and
```
assertEquals(40, allowedNameWithMaxLength.length());
```

to remove the distraction in code.